course6/toyapp_alexs/displaylib/vector.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <array>
+
+namespace Display
+{
+	template<class T, std::size_t S>
+	struct Vector
+	{
+	};
+
+	template<>
+	struct Vector<int, 2>
+	{
+		int x;
+		int y;
+	};
```

Comment:
To complete this taks, you should add some useful methods to the class: arithmetic operators, equality operators, streaming operator etc.